docs/source/configuration/key_value_service_configs/oracle_key_value_service_config.rst

```diff
@@ -198,34 +198,70 @@ migration as `serviceNameConfiguration.namespaceOverride`.
 
-To resolve, simply add the following configuration to your Oracle ddl configuration.
+To resolve, simply add the following configuration to your Oracle ddl
+configuration.
 
-.. code-block:: yaml
+.. code:: yaml
 
-    atlasdb:
-      keyValueService:
-        type: relational
-        ddl:
-          type: oracle
-          alterTablesOrMetadataToMatchAndIKnowWhatIAmDoing:
-              - namespace:
-                  name: <namespace>
-                tablename: <table-name>
+   atlasdb:
+     keyValueService:
+       type: relational
+       ddl:
+         type: oracle
+         alterTablesOrMetadataToMatchAndIKnowWhatIAmDoing:
+             - physicalTableName: <physicalTableName>
+
+..
+.. tip::
+
+   To determine your ``physicalTableName`` given an ``ORA-00904``
+   stacktrace, you should see a table name with a similar form to ``x_yz__tablename_1234`` within the
+   full SQL string that was ran. The full string ``x_yz__tablename_1234`` is your ``physicalTableName``.
+
+.. tip::
+
+   If you already know the logical table name and table namespace that's
+   affected (e.g, using your AtlasDB schema file), then you can use the
+   following configuration instead:
+
+   ::
 
-Note: Table names are case sensitive.
+      atlasdb:
+       keyValueService:
+         type: relational
+         ddl:
+           type: oracle
+           alterTablesOrMetadataToMatchAndIKnowWhatIAmDoing:
+               - namespace:
+                   name: <namespace>
+                   tablename: <table-name>
 
 Generally speaking the operation is safe to perform, although it's on the operator to determine what the side
-effects are. For example, if the issue arose as two services are configured to use this table, but only one is
-performing table mapping, then it is expected that this could break one of the services. However, that condition
-still satisfies the status quo, thus it's on the configurator to determine if this change is safe to make.
+effects are. For example, if the issue arose as two services are configured to use this table, but only
+one is performing table mapping, then it is expected that this could break one of the services. However,
+that condition still satisfies the status quo, thus it's on the configurator to determine if this change is
+safe to make.
 
 Although the alter action is idempotent, it is recommended to remove the configuration after it has ran. Check the logs
-for ``Altering table to have overflow column to match metadata.`` and that there were no error logs printed to ensure that
-has successfully ran.
\ No newline at end of file
+for ``Altering table to have overflow column to match metadata.`` and that there were no error logs printed to ensure
+that has successfully ran.
+
+If things do not work:
+
+#. Check for log lines containing ``Potentially altering table {} (internal name: {}) to have overflow column.``, and verify that your table reference / physical table name shows up in one of the log lines.
```

Comment:
In general: do we want to include the slf4j placeholders in this kind of thing? They will work in internal log search product, but might not in some of our other use cases...